zeek-ocsf/package.yaml

```diff
@@ -567,278 +626,114 @@ pipelines:
       }
       subscribe "zeek"
       where @name == "zeek.ssh"
-      // The Authentication Type can be deduced if we have the `inferences`
-      // package.
-      if "KS" in event.inferences {
-        auth_type_id = 4
-        auth_type = "Keyboard Interactive"
-      } else {
-        auth_type = 0
-        auth_type = "Unknown"
-      }
-      this = { event: this }
-      class_uid = 4007
+      this = { zeek: this }
+      // === Classification ===
       // The activity attributes relate to the underlying TCP connection. Since
       // we have a ssh.log entry, it means we already have an established
       // connection and must assume data is being exchanged, even if it's just
       // the SSH handshake.
-      activity_id = 6
-      activity_name = "Traffic"
-      // The Authentication Type can be deduced if we have the `inferences`
-      // package.
-      if "KS" in event.inferences {
-        auth_type_id = 4
-        auth_type = "Keyboard Interactive"
-      } else {
-        auth_type_id = 0
-        auth_type = "Unknown"
-      }
-      // Map direction.
-      // The activity attributes relate to the underlying TCP connection. Since
-      // we have a ssh.log entry, it means we already have an established
-      // connection and must assume data is being exchanged, even if it's just
-      // the SSH handshake.
-      direction = $direction[event.direction].name
-      direction_id = $direction[event.direction].id
-      // Map network layer protocol version.
-      if event.id.orig_h.is_v6() or event.id.resp_h.is_v6() {
-        protocol_ver_id = 6
-      } else {
-        protocol_ver_id = 4
-      }
-      // Adjust the severity.
-      if "ABP" in event.inferences or "BFS" in event.inferences {
+      ocsf.activity_id = 6
+      ocsf.activity_name = "Traffic"
+      ocsf.category_uid = 4
+      ocsf.category_name = "Network Activity"
+      ocsf.class_uid = 4007
+      ocsf.class_name = "SSH Activity"
+      ocsf.type_uid = ocsf.class_uid * 100 + ocsf.activity_id
+      if "ABP" in zeek.inferences or "BFS" in zeek.inferences {
         // Client authentication bypass (ABP) or brute force success (BFS) are
         // worthwhile escalating.
-        severity_id = 4
-        severity = "High"
-      } else if "BF" in event.inferences \
-        or "SV" in event.inferences \
-        or "SC" in event.inferences \
-        or "SC" in event.inferences {
+        ocsf.severity_id = 4
+        ocsf.severity = "High"
+      } else if "BF" in zeek.inferences \
+        or "SV" in zeek.inferences \
+        or "SC" in zeek.inferences \
+        or "SC" in zeek.inferences {
```

Comment:
I would not use `\` here. I think putting parenthesis around everything is better here, or ending the line with `or` should also work.